controller/ControllerRTL.py

```diff
@@ -10,51 +10,58 @@
 
 from pymtl3 import *
 from pymtl3.stdlib.primitive import RegisterFile
-from ..lib.basic.val_rdy.ifcs import SendIfcRTL as SendIfcRTL
-from ..lib.basic.val_rdy.ifcs import RecvIfcRTL as RecvIfcRTL
+from ..lib.basic.en_rdy.ifcs import SendIfcRTL, RecvIfcRTL

```

Comment:
I think you need to sync to latest master, we already deprecate the en_rdy ifcs.